addon/components/inputs/checkbox-array.js

```diff
@@ -0,0 +1,47 @@
+import AbstractInput from './abstract-input'
+import computed, {readOnly} from 'ember-computed-decorators'
+import layout from 'ember-frost-bunsen/templates/components/frost-bunsen-input-checkbox-array'
+
+export default AbstractInput.extend({
+  // == Component Properties ===================================================
+
+  classNames: [
+    'frost-bunsen-input-checkbox-array',
+    'frost-field'
+  ],
+
+  layout,
+
+  // == State Properties =======================================================
+
+  selected: [],
+
+  // == Computed Properties ====================================================
+
+  @readOnly
+  @computed('bunsenModel')
+  options (bunsenModel) {
+    return bunsenModel.items.enum.map((key) => {
+      return {
+        value: key
+      }
+    })
+  },
+
+  // == Functions ==============================================================
+  /**
+   * This should be overriden by inherited inputs to convert the value to the appropriate format
```

Comment:
This description only makes sense on the `AbstractInput`. Here I'd change it to say something like, "Convert value to appropriate format for template".